Gemfile

```diff
@@ -47,6 +47,7 @@ group :development, :test do
   gem 'byebug'
   gem 'better_errors'
   gem 'binding_of_caller'
+  gem 'meta_request'
```

Comment:
also added meta_request to see more information in inspect element when using chrome browser